packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -67,10 +67,18 @@ class WebpackCLI {
     }
 
     // For babel/typescript
-    if (result.default) {
+    if (result && result.default) {
       result = result.default;
     }
 
+    // Treat undefined configuration for zero-config
+    if (!result) {
+      this.logger.info(
+        "Configuration file provided didn't export any configuration, using defaults to build.",
+      );
```

Comment:
Let's avoid it, too aggressive logging, empty nothing = empty configuration, no need to logging about it

Reply:
Okay, will remove.